### PR TITLE
Cleaned up docblock comment formatting

### PR DESCRIPTION
**Pull request by samtuke**

### CODE REVIEW

File: public_html/lists/admin/defaultplugin.php [hunk 1]
```diff
@@ -249,19 +258,22 @@ public function pageTitleHover($page)
...
+     * deleteSent - wipe DB entries marking a campaign sent for subscribers
      * this is used in DEV mode only.
      */
     public function deleteSent()
     {
     }
 
+    /** 
+     * write a value to the general config to be retrieved at a later stage
+     * parameters: name -> name of the variable
```
Comment (oliverklee) [inline public_html/lists/admin/defaultplugin.php]: These should use the "param" PHPDoc annotation.

File: public_html/lists/admin/defaultplugin.php [hunk 2]
```diff
@@ -275,12 +287,13 @@ public function writeConfig($name, $value)
         return 1;
     }
 
+    /** 
+     * read a value from the general config to be retrieved at a later stage
+     * parameters: name -> name of the variable
```
Comment (oliverklee) [inline public_html/lists/admin/defaultplugin.php]: These should use the "param" and "return" PHPDoc annotations.

File: public_html/lists/admin/defaultplugin.php [hunk 3]
```diff
@@ -299,13 +312,15 @@ public function getConfig($name)
         return $result;
     }
 
+    /** 
+     * displayConfig
+     * purpose: display input for a config variable in the backend
+     * parameters:
+     * name -> name of the config variable, as found in $this->configvars
+     * return, HTML snippet of input to slot into a form
```
Comment (oliverklee) [inline public_html/lists/admin/defaultplugin.php]: These should use the "param" and "return" PHPDoc annotations.

File: public_html/lists/admin/defaultplugin.php [hunk 4]
```diff
@@ -373,14 +390,16 @@ public function validateSubscriptionPage($pageData)
         return;
     }
 
+    /** 
+     * parse the text of the thankyou page
+     * parameters:
+     * pageid -> id of the subscribe page
+     * userid -> id of the user
+     * text -> current text of the page
+     * returns parsed text
```
Comment (oliverklee) [inline public_html/lists/admin/defaultplugin.php]: These should use the "param" and "return" PHPDoc annotations.

File: public_html/lists/admin/defaultplugin.php [hunk 5]
```diff
@@ -573,75 +598,79 @@ public function processSendingCampaignFinished($messageid, array $messagedata)
...
-        // );
         return array();
     }
 
+    /**
+     * mimeWrap
+     * purpose: wrap the actual contents of the message in another MIME layer
+     * Designed to ENCRYPT the fully expanded message just before sending
+     * Designed to be called by phplistmailer
+     * parameters:
```
Comment (oliverklee) [inline public_html/lists/admin/defaultplugin.php]: These should use the "param" and "return" PHPDoc annotations.

File: public_html/lists/admin/defaultplugin.php [hunk 6]
```diff
@@ -573,75 +598,79 @@ public function processSendingCampaignFinished($messageid, array $messagedata)
...
     }
 
+    /**
+     * setFinalDestinationEmail
+     * purpose: change the actual recipient based on user Attribute values:
+     * parameters:
+     *   messageid: message being sent
+     *   uservalues: array of "attributename" => "attributevalue" of all user attributes
+     *   email: email that this message is current set to go out to
+     * @returN email that it should go out to
```
Comment (oliverklee) [inline public_html/lists/admin/defaultplugin.php]: These should use the "param" and "return" PHPDoc annotations.

Comment (samtuke) [inline public_html/lists/admin/defaultplugin.php]: Yes, you're absolutely right, however there's only so much manual reviewing and fixing I can do in one afternoon :) These commits improve what was there; they do not perfect it.

Comment (oliverklee) [inline public_html/lists/admin/defaultplugin.php]: Okay then.

**Pull request merged**